models/multiphase/d2q9_pf_pressureEvolution/Dynamics.R

```diff
@@ -58,8 +58,8 @@ AddSetting(name="sigma", 		   comment='surface tension')
 # 	Inputs: Fluid Properties
 AddSetting(name="omega_l", comment='one over relaxation time (low density fluid)')
 AddSetting(name="omega_h", comment='one over relaxation time (high density fluid)')
-AddSetting(name="nu_l", omega_l='1.0/(3*nu_l)', default=0.16666666, comment='kinematic viscosity')
-AddSetting(name="nu_h", omega_h='1.0/(3*nu_h)', default=0.16666666, comment='kinematic viscosity')
+AddSetting(name="Viscosity_l", omega_l='1.0/(3*Viscosity_l)', default=0.33333333, comment='kinematic viscosity')
+AddSetting(name="Viscosity_h", omega_h='1.0/(3*Viscosity_h)', default=0.33333333, comment='kinematic viscosity')
```

Comment:
I had just put as 1/3 quite awhile ago, I think I had been just doing it to set omega to 1. But sorry, I agree 1/6 is more reasonable as corresponds to 1.0 relaxation. I will change and re-commit.